tests/test_client_regress/views.py

```diff
@@ -109,6 +109,10 @@ def return_json_response(request):
     return JsonResponse({'key': 'value'})
 
 
+def return_json_vendor_response(request):
```

Comment:
You could modify the existing `return_json_response` view to extra `content_type` from `request.GET` to allow more tests without creating a separate view for each one.